index.html

```diff
@@ -1129,57 +1089,58 @@ <h1>Data Model</h1>
+as specified in the [[[INFRA]]].
           </td>
         </tr>
         <tr>
           <td>
 <dfn>integer</dfn>
           </td>
           <td>
-A real number without a fractional component as specified in
-[<a data-cite="XMLSCHEMA11-2#decimal">XMLSCHEMA11-2</a>]. To maximize
-interoperability, implementers are urged to heed the advice regarding
-integers in <a data-cite="rfc8259#section-6">RFC8259, Section 6: Numbers</a>.
+A <a data-cite="XMLSCHEMA11-2#decimal">real number without a fractional
+component</a> as specified in [[XMLSCHEMA11-2]]. To maximize interoperability,
+implementers are urged to heed the advice regarding integers in
+<a data-cite="rfc8259#section-6">RFC8259, Section 6: Numbers</a>.
           </td>
         </tr>
         <tr>
           <td>
 <dfn>double</dfn>
           </td>
           <td>
-A value that is often used to approximate arbitrary real numbers as specified
-in [<a data-cite="XMLSCHEMA11-2#double">XMLSCHEMA11-2</a>]. To maximize
-interoperability, implementers are urged to heed the advice regarding
-doubles in <a data-cite="rfc8259#section-6">RFC8259, Section 6: Numbers</a>.
+A <a data-cite="XMLSCHEMA11-2#double">real number with a fractional
+component</a> that is often used to approximate arbitrary real numbers as
+specified in [[XMLSCHEMA11-2]]. To maximize interoperability, implementers are
+urged to heed the advice regarding doubles in
+<a data-cite="rfc8259#section-6">RFC8259, Section 6: Numbers</a>.
           </td>
         </tr>
         <tr>
           <td>
 <a data-cite="INFRA#boolean">boolean</a>
           </td>
           <td>
-A value that is either true or false as defined in [[INFRA]].
+A value that is either true or false as defined in the [[[INFRA]]].
           </td>
         </tr>
         <tr>
           <td>
 <a data-cite="INFRA#nulls">null</a>
           </td>
           <td>
-A value that is used to indicate the lack of a value as defined in [[INFRA]].
+A value that is used to indicate the lack of a value as defined in the [[[INFRA]]].
           </td>
         </tr>
       </tbody>
     </table>
 
     <p class="advisement" title="Ordering of values">
 As a result of the <a href="#data-model">data model</a> being defined using
-terminology from [[INFRA]], property values which can contain more than one
-item, such as <a data-cite="INFRA#list">lists</a>, <a
+terminology from the [[[INFRA]]], property values which can contain more than
+one item, such as <a data-cite="INFRA#list">lists</a>, <a
 data-cite="INFRA#ordered-map">maps</a> and <a
 data-cite="INFRA#ordered-set">sets</a>, are explicitly ordered. All list-like
-value structures in [[INFRA]] are ordered, whether or not that order is
+value structures in the [[[INFRA]]] are ordered, whether or not that order is
 significant. For the purposes of this specification, unless otherwise stated, <a
 data-cite="INFRA#ordered-map">map</a> and <a
 data-cite="INFRA#ordered-set">set</a> ordering is not important and
```

Comment:
```suggestion
significant. For the purposes of this specification, unless otherwise stated,
ordering of a <a data-cite="INFRA#ordered-map">map</a> or <a
data-cite="INFRA#ordered-set">set</a> is not important and
```